plugin/trino-smartnews-functions/src/test/java/io/trino/plugin/smartnews/functions/TestHashFunctions.java

```diff
@@ -72,11 +74,28 @@ public void testHashConsistency() throws Exception
     }
 
     @Test
-    public void testDynamics() throws Exception
+    public void testDynamicHashing() throws Exception
     {
-        String prevRes = runner.execute("select sn_hash('123')").getOnlyValue().toString();
+        Session session = TestingSession.testSessionBuilder().build();
+        String prevRes = runner.execute(session, "select sn_hash('123')").getOnlyValue().toString();
+        init();
```

Comment:
I don't know why, the init() could occasionally cause error: sn_hash is not registered.
So I guess you can initialized another session object, and use different session object to achieve your goal, without calling init() again.

Reply:
fixed